README.md

```diff
@@ -437,31 +429,27 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   end
   ~~~
 
-* Avoid hashes-as-optional-parameters in general. Does the method do too much?
-
 * Prefer keyword arguments over options hash.
 
 * Prefer `map` over `collect`, `find` over `detect`, `select` over `find_all`,
   `size` over `length`.
 
-* Prefer `Time` over `DateTime` since it supports proper time zones instead of
-  UTC offsets. [More
-  info](https://gist.github.com/pixeltrix/e2298822dd89d854444b).
+* Prefer `Time` over `DateTime`.
```

Comment:
Imo saying TZ vs offsets provides a nice and short rational

Reply:
With this change I'm removing such bits of rationale from rules to make them more concise. The fact that `Time` supports time zones is not the reason why people should follow the rule. It's because it makes code more consistent (like all the other rules).